easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -340,4 +345,8 @@ private static void checkCallingObjectSuitability(Object object) {
             throw new IllegalArgumentException("Caller must implement PermissionCallbacks.");
         }
     }
+
+    private static boolean isUsingAndroidAnnotations(Class clazz) {
+        return clazz.getSimpleName().endsWith("_") && HasViews.class.isAssignableFrom(clazz);
```

Comment:
Rather than adding androidannotations as a provided dep, we can use the string name of the class in this check.

See: http://stackoverflow.com/a/2170543/324977